.betterer.results

```diff
@@ -4974,6 +4969,11 @@ exports[`better eslint`] = {
       [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "7"],
       [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "8"]
     ],
+    "public/app/features/plugins/admin/components/PluginDetailsRightPanel.tsx:5381": [
```

Comment:
I know you are moving things around but maybe a good chance to get rid of these linting issues?